### PR TITLE
refactor(react): whatsappTemplate

### PR DESCRIPTION
**Pull request by AinaVendrell**
## Description
Adapt WhatsappTempletes and create a new message type named `whatsapptemplate`.

## Context
WhatsappTempletes were send as text

## Approach taken / Explain the design
If rendered in the browser it shows a message with the name and the spacename, if rendered in node it sends all properties to the message with whatsapptemplate message type

## To document / Usage example
`WhatsappTemplate` component allows to send whatsapp templates in the bot actions

Properties:
required: name, namespace, language
non-required: header, body, footer

Example:
In `./actions/example-whatsappTemplate.js`
```
import React from 'react'
import { WhatsappTemplate } from '@botonic/react'

export default class extends React.Component {
  render() {
    return (
      <WhatsappTemplate
        name={'out_of_office'}
        namespace={'d32863cc_3dd6_3e84_74c0_38d758fc60f6'}
        language={'en'}
        header={''}
        body={{
          type: 'body',
          parameters: [{ type: 'text', text: 'Anthony' }],
        }}
        footer={''}
      ></WhatsappTemplate>
    )
  }
}
```

## Testing

The pull request...

- [ ] has unit tests
- [ ] has integration tests
- [ ] doesn't need tests because... **[provide a description]**

### CODE REVIEW

**Comment by dpinol** [comment]
Please remove "
Set as Draft PR if it's not ready to be merged.

PR best practices Reference"
when creating new PRs

File: packages/botonic-core/src/index.d.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const INPUT: {
   CAROUSEL: 'carousel'
   CUSTOM: 'custom'
   WEBCHAT_SETTINGS: 'webchatsettings'
+  WHATSAPP_TEMPLATE: 'whatsapptemplate'
```
Comment (dpinol) [inline packages/botonic-core/src/index.d.ts:28]: thanks for also updating the typescript types!

File: packages/botonic-react/src/components/whatsapp-template.jsx [hunk 2]
```diff
@@ -3,32 +3,23 @@ import { Message } from './message'
...
+}
+
 export const WhatsappTemplate = props => {
   const renderBrowser = () => {
-    let params = ''
-    for (const param in props.parameters) {
-      params = params + " '" + props.parameters[param] + "', "
-    }
     // Return a dummy message for browser
+    const message = `Template ${props.name} with namespace ${props.namespace} would be send to the user.`
```
Comment (dpinol) [inline packages/botonic-react/src/components/whatsapp-template.jsx:13]: send-> sent

**Review by dpinol — APPROVED** [review]

**Review by vanbasten17 — APPROVED** [review]
By looking at the example, I'm wondering...

```
<WhatsappTemplate
        name={'out_of_office'}
        namespace={'d32863cc_3dd6_3e84_74c0_38d758fc60f6'}
        language={'en'}
        header={''}
        body={
          '{"type": "body","parameters": [{"type": "text","text": "Anthony"}]}'
        }
        footer={''}
      ></WhatsappTemplate>
```

Is it possible to pass the props as objects for `header`, `body` and `footer` instead of a stringified version of an object? I think it will facilitate the work for us and the developers. You can let the dev pass an object and stringify it within the WhatsappTemplate component. For the rest, LGTM 👍

**Pull request merged**